tests/integration/test_ourlogs.py

```diff
@@ -77,7 +79,9 @@ def test_ourlog_extraction(
         "project_id": 42,
         "retention_days": 90,
         "timestamp_nanos": int(start.timestamp() * 1e9),
-        "observed_timestamp_nanos": int(end.timestamp() * 1e9),
+        "observed_timestamp_nanos": time_within_delta(
+            start.timestamp(), expect_resolution="ns"
```

Comment:
Maybe a silly question but is there a reason we do "ns" here rather than `Resolution.NanoSeconds` ?

Reply:
Just for convenience, both work.